src/multi_vector_simulator/E3_indicator_calculation.py

```diff
@@ -876,14 +879,49 @@ def equation_onsite_energy_matching(
     return onsite_energy_matching
 
 
-def equation_co2_emissions(dict_values):
-    co2_emissions = 0
-    for asset in dict_values[ENERGY_PRODUCTION]:
-        co2_emissions += (
-            dict_values[ENERGY_PRODUCTION][asset]["total_aggregated_flow"][VALUE]
-            * dict_values[ENERGY_PRODUCTION][asset]["emissionFactor"][VALUE]
-        )
-    return co2_emissions
+def add_total_emissions(dict_values):
+    r"""
+    Calculates the total emission of the energy system in kg/a and kg/kWheleq and adds KPI to `dict_values`.
+
+    Parameters
+    ----------
+    dict_values: dict
+        All simulation inputs and results
+
+    Returns
+    -------
+    None
+        updated `dict_values` with total emissions KPI, unit: kg/a and kg/kWheleq
+
+    Notes
+    -----
+    Tested with:
+    - ()
+
+    """
+    # sum up emissions of all assets [kg/a]
+    emissions = dict_values[KPI][KPI_SCALAR_MATRIX][TOTAL_EMISSIONS].sum()  # data frame
+    dict_values[KPI][KPI_SCALARS_DICT].update({TOTAL_EMISSIONS: emissions})
+    logging.debug(f"Calculated the {TOTAL_EMISSIONS}: {round(emissions, 2)} kg/a.")
+    logging.info(f"Calculated the {TOTAL_EMISSIONS} kg/a of the LES.")
+
+    # emissions per kWheleq
+    emissions_kWh = (
+        emissions
+        / dict_values[KPI][KPI_SCALARS_DICT][
+            TOTAL_DEMAND + SUFFIX_ELECTRICITY_EQUIVALENT
+        ]
+    )
+    dict_values[KPI][KPI_SCALARS_DICT].update(
+        {TOTAL_EMISSIONS + SUFFIX_ELECTRICITY_EQUIVALENT: emissions_kWh}
```

Comment:
I think I do not like the name `Total emissions electricity equivalents`. Maybe we introduce a new one? `Specific emissions per electricity equivalent`